src/SDK/Common/Util/TracingUtil.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Common\Util;
+
+use function array_key_exists;
+use function array_shift;
+use function basename;
+use function in_array;
+use function sprintf;
+use function str_replace;
+
+use Throwable;
+
+class TracingUtil
```

Comment:
Could you please rename the class to smth like `StackTraceFormatter` and the method to just `format`, please?